apps/web/src/lib/seo/generators/structured-data.tsx

```
@@ -0,0 +1,172 @@
+      }
+      break;
+  }
+
+  return (
+    <>
+      {Object.entries(schemas).map(([key, schema]) => (
+        <Script
+          key={`structured-data-${key}`}
+          id={`structured-data-${key}`}
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)